Gordon360/Services/RecIM/TeamService.cs

```diff
@@ -344,8 +349,47 @@ public async Task<TeamViewModel> UpdateTeamAsync(int teamID, TeamPatchViewModel
 
             return t;
         }
+
+        private async Task SendInviteEmail(int teamID, string inviteeUsername, string inviterUsername)
+        {
+            var team = _context.Team.FirstOrDefault(t => t.ID == teamID);
+            var activity = _context.Activity.FirstOrDefault(a => a.ID == team.ActivityID);
+            var invitee = _accountService.GetAccountByUsername(inviteeUsername);
+            var inviter = _accountService.GetAccountByUsername(inviterUsername);
+
+            string from_email = _config["Emails:Sender:Username"];
+            string to_email = invitee.Email;
+            string messageBody =
+                 $"Hey {invitee.FirstName}!<br><br>" +
+                $"{inviter.FirstName} {inviter.LastName} has invited you join <b>{team.Name}</b> for <b>{activity.Name}</b> <br>" +
+                $"Registration closes on <i>{activity.RegistrationEnd.ToString("D", CultureInfo.GetCultureInfo("en-US"))}</i> <br>" +
+                //$"check it out <a href='https://360.gordon.edu/recim'>here</a>! <br><br>" + //for production
+                $"check it out <a href='https://360recim.gordon.edu/recim'>here</a>! <br><br>" +//for development
```

Comment:
Can you use the server util function for getting the web address here? That would allow us to distinguish between train, production, and local development.

Reply:
> Can you use the server util function for getting the web address here? That would allow us to distinguish between train, production, and local development.

I forgot that existed, I'll implement that!

Reply:
@bennettforkner I actually don't know how to use ServerUtil, and there isn't a reference on 360. For now, I'll probably just keep it the way it is or have the url be a part of config

Reply:
@amos just `_serverUtils.GetAddress()` will give you the right thing.

Reply:
server utils do not exist in the rec-im branch
we will need to do a merge

Reply:
You could do a cherry-pick if you can find the right commits. But yes, something you can add an issue for to complete after you merge in develop.

Reply:
Either way, can you use the environment here rather than having code that has to be uncommented every time we move to production?

Reply:
@bennettforkner  When I mean production, I mean when recim is deployed onto 360's develop

Reply:
Then I guess that begs the question, why even make it nullable, which for that. I have no clue. I'll make that adjustment.

Reply:
> @bennettforkner  When I mean production, I mean when recim is deployed onto 360's develop, when recim exists on the actual 360 website, we won't need to refer back to the "develop" line and more importantly ServerUtils should exist by then.

Well in develop we would want the link to point to 360train.gordon.edu, not 360.gordon.edu. So really, you would really want three URLs here and two of them would be commented out. Or you could do it the better way and take it from the appsettings file.

Either way, it's not super important now of course, but you'll have to remember to use serverutils here when you get develop merged in.